Hearthstone Deck Tracker/Controls/CardToolTipControl.xaml

```diff
@@ -38,7 +39,11 @@
             <TextBlock Name="CardTextBlock" Text="{Binding Text}" Grid.Row="1" FontSize="14"
                        Foreground="{DynamicResource TextBrush}" VerticalAlignment="Top" FontWeight="Medium"
                        HorizontalAlignment="Center" TextWrapping="Wrap" Margin="10,0,10,0" TextAlignment="Center" />
-            <Grid Grid.Row="2" Background="{DynamicResource AccentColorBrush}">
+            <TextBlock Name="AlternativeLanguageText" Text="{Binding AlternativeLanguageText}" Grid.Row="2" FontSize="12"
+                       Foreground="#FF7F7F7F" VerticalAlignment="Top"
+                       Visibility="{Binding ShowAlternativeLanguageTextInTooltip}"
```

Comment:
I tried to hide the TextBlock showing alternative texts by setting Visibility property here. However it does not work as expected. Do you have some idea?